lib/lazy_load_indexed_stack.dart

```diff
@@ -45,26 +45,33 @@ class LazyLoadIndexedStack extends StatefulWidget {
 class _LazyLoadIndexedStackState extends State<LazyLoadIndexedStack> {
   late final List<bool> _alreadyLoaded;
 
+  late List<Widget> _children;
+
+  final stackKey = GlobalKey();
+
   @override
   void initState() {
     super.initState();
 
     _alreadyLoaded = List.filled(widget.children.length, false);
     _alreadyLoaded[widget.index] = true;
+
+    _children = _loadedChildren();
   }
 
   @override
   void didUpdateWidget(final LazyLoadIndexedStack oldWidget) {
-    super.didUpdateWidget(oldWidget);
-
     _alreadyLoaded[widget.index] = true;
+    _children[widget.index] = widget.children[widget.index];
+    super.didUpdateWidget(oldWidget);
```

Comment:
`super.didUpdateWidget(oldWidget);` should be called first.

> Implementations of this method should start with a call to the inherited method, as in super.didUpdateWidget(oldWidget).

ref: https://api.flutter.dev/flutter/widgets/State/didUpdateWidget.html